outlook/mail/credential/tool.gpt

```diff
@@ -4,6 +4,7 @@ Share Credential: ../../../oauth2 as outlook.mail.read
         microsoft365 as integration and
         "Mail.Read
         User.Read
+        MailboxSettings.Read
```

Comment:
Just curious: does this mean that any existing credentials won't work? Or does this change of scope get recognized and the credential re-requested?

Reply:
Correct. Unfortunately, credentials will have to be manually deleted and recreated. GPTScript's credentials framework does not store information about OAuth scopes.